app/models/organization.rb

```diff
@@ -53,4 +57,12 @@ def ensure_url
       errors.add(:web, :url_format_invalid)
     end
   end
+
+  def display_id(destination_accountable)
+    if destination_accountable.is_a?(Organization)
```

Comment:
There are quite a lot of this type checking on Organization. Very smelly :nose: so I guess this should be our next focus.